lib/SIL/OwnershipUtils.cpp

```diff
@@ -43,6 +43,7 @@ bool swift::isOwnershipForwardingValueKind(SILNodeKind kind) {
   case SILNodeKind::CondBranchInst:
   case SILNodeKind::DestructureStructInst:
   case SILNodeKind::DestructureTupleInst:
+  case SILNodeKind::MarkDependenceInst:
```

Comment:
This function (and isGuaranteedForwardingValue) should have a doc comment saying that it returns true if the resulting value has been forwarded from an incoming value. For `isOwnership|GuaranteedForwardingInst`, we don't know which operand it was forwarded from.

Reply:
I can add that. At a higher level, I want to somehow tie this to the adding of a new ownership forwarding instruction. Creating this ad hoc rule is called "lets get this code out of sync with everything else ASAP please!"